Kudu.Services/SiteExtensions/SiteExtensionController.cs

```diff
@@ -233,14 +233,9 @@ public SiteExtensionController(ISiteExtensionManager manager, IEnvironment envir
                 ITracer backgroundTracer = NullTracer.Instance;
                 IDictionary<string, string> traceAttributes = new Dictionary<string, string>();
 
-                if (tracer.TraceLevel == TraceLevel.Off)
-                {
-                    backgroundTracer = NullTracer.Instance;
-                }
-
                 if (tracer.TraceLevel > TraceLevel.Off)
                 {
-                    backgroundTracer = new XmlTracer(_environment.TracePath, tracer.TraceLevel);
+                    backgroundTracer = new CascadeTracer(new XmlTracer(_environment.TracePath, tracer.TraceLevel), new ETWTracer(_environment.RequestId, "PUT"));
```

Comment:
No, that's fine, just making sure it wasn't some accidental code that wasn't meant to go in.